book/src/powdr_vm_intro.md

```diff
@@ -0,0 +1,13 @@
+# Introduction
+powdrVM is a versatile zkVM built on the [powdr](./powdr_sdk_intro.md) stack, supporting std Rust, multiple proof systems, and zk-continuations for unbounded execution. Designed for flexibility, powdrVM enables efficient proof generation for complex applications with minimal setup.
```

Comment:
```suggestion
powdrVM is a versatile zkVM built on the [powdr](./powdr_sdk_intro.md) stack, supporting std Rust,
multiple proof systems, and zk-continuations for unbounded execution.
Designed for flexibility, powdrVM enables efficient proof generation for complex applications with minimal setup.
```
just for ease of PR reviews. Not sure if this renders into an actual newline, but you did the same below.

Reply:
isn't it the opposite, that single lines are easier for reviews because the diff is more accurate and doesn't need to understand line breaks?

Reply:
If you don't change the position of line breaks, the diff is much easier to read for me.

Reply:
ok